Assets/HoloToolkit/UX/Scripts/UICollection.cs

```diff
@@ -63,9 +63,9 @@ private void Awake()
         private void Start()
         {
             // Verify this is attached to a GameObject with a rect transform
-            m_rectTransform = GetComponent<RectTransform>();
+            rectTransform = GetComponent<RectTransform>();
 
-            if (m_rectTransform == null)
+            if (rectTransform == null)
             {
                 Debug.LogError("This component must be attached to a GameObject with a RectTransform component!");
```

Comment:
Do we really need this debug log if the component is required?

Reply:
We don't